app/models/report_generators/system_performance/fy2019/base.rb

```diff
@@ -26,6 +26,7 @@ def add_filters(scope:)
         scope = scope.joins(:project).where(Project: { id: project_ids})
       end
       scope = scope.where(data_source_id: @report.options['data_source_id'].to_i) if @report.options['data_source_id'].present?
```

Comment:
Should this be removed?

Reply:
I was leaving it.  We're removing the entire SPM shortly, and this doesn't cause any harm.